package.json

```diff
@@ -179,6 +180,7 @@
     "nodemailer-mailgun-transport": "1.4.0",
     "omit-deep-lodash": "1.1.4",
     "onefx": "1.8.5",
+    "piexifjs": "^1.0.6",
```

Comment:
piexifjs is too large for the client side bundle

let's just do not fix this issue